packages/portalnetwork/src/client/client.ts

```diff
@@ -1148,7 +1170,9 @@ export class PortalNetwork extends (EventEmitter as { new (): PortalNetworkEvent
         return
       }
     }
-    this.client.sendTalkResp(enr, message.id, payload)
+    const dstPeerId = await enr.peerId()
+    const dstAddr = new Multiaddr(src.socketAddr.toString() + '/ipfs/' + dstPeerId.toB58String())
```

Comment:
Is this a workaround for the ENR verification check?  Or is this part of the rendezvous setup?

Reply:
It's a workaround and I discovered it doesn't work that week with further changes I've made on the discv5 side so we have more work to do anyway.

Reply:
This should be resolved now, though it's just as hacky as this.